frontend/src/slices/auth/actions.ts

```diff
@@ -51,6 +55,50 @@ const signUp = createAsyncThunk<
   },
 );
 
+const editCustomer = createAsyncThunk<
+  CustomerEditDto,
+  CustomerEditDto,
+  AsyncThunkConfig<HttpError>
+>(`${sliceName}/edit-customer`, async (payload, { extra, rejectWithValue }) => {
+  const { userApi, notification } = extra;
+
+  try {
+    const editedCustomer = await userApi.editCustomer(payload);
+
+    notification.success(UserNotificationMessage.SUCCESS_EDIT_USER);
+
+    return editedCustomer;
+  } catch (error_: unknown) {
+    const error = error_ as HttpError;
+
+    notification.error(getErrorMessage(error.message));
```

Comment:
It would be nice to see an error message below the specific field in the form of a parameter is invalid so that we don't have to close the form and re-enter the same data. Notifications should be provided only for system errors or errors in the completed action.